berkeley-mobile/MainContainerViewController.swift

```diff
@@ -47,6 +49,17 @@ class MainContainerViewController: UIViewController, MainDrawerViewDelegate {
         drawerStatePositions[.full] = self.view.safeAreaInsets.top + (self.view.frame.maxY / 2)
         self.initialDrawerCenter = drawerViewController!.view.center
         moveDrawer(to: drawerViewController!.currState, duration: 0)
+        
+        let launchedBefore = UserDefaults.standard.bool(forKey: "launchedBefore")
+        if launchedBefore  {
+           print("Not first launch")
+        } else {
+            let vc = PageViewController(transitionStyle: .scroll, navigationOrientation: .horizontal)
+            vc.modalPresentationStyle = .fullScreen
+            self.present(vc, animated: true, completion: nil)
+            print("First launch")
+            UserDefaults.standard.set(true, forKey: "launchedBefore")
```

Comment:
change the name of the key to something like "hasShownStudyPact" because we may want to use this sort of thing for future feature onboardings. "launchedBefore" is a bit ambiguous/general

Reply:
also you can remove the prints